velox/core/Expressions.h

```diff
@@ -417,10 +410,15 @@ class LambdaTypedExpr : public ITypedExpr {
   }
 
   TypedExprPtr rewriteInputNames(
-      const std::unordered_map<std::string, std::string>& mapping)
+      const std::unordered_map<std::string, TypedExprPtr>& mapping)
       const override {
+    for (const auto& name : signature_->names()) {
```

Comment:
Do we have unit test for this?

Reply:
This shouldn't happen, because presto will write lambas with different names. There is an e2e test in presto cpp